scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -1763,18 +1778,20 @@ function Set-ProvisioningMode {
                     "    iothub_hostname: '$IotHubHostName'"
                     "    device_id: '$DeviceId'"
                     "    identity_cert: '$certUri'"
-                    "    identity_pk: '$pkUri'")
+                    "    identity_pk: '$pkUri'",
+                    "  dynamic_reprovisioning: $DynamicReprovisioning")
             }
             $configurationYaml = ($configurationYaml -replace $selectionRegex, ($replacementContent -join "`n"))
             Write-HostGreen 'Configured device for manual provisioning.'
             return $configurationYaml
         }
         elseif ($External -or $ExternalProvisioningEndpoint){
-            $selectionRegex = '(?:[^\S\n]*#[^\S\n]*)?provisioning:\s*#?\s*source:\s*".*"\s*#?\s*endpoint:\s*".*"'
+            $selectionRegex = '(?:[^\S\n]*#[^\S\n]*)?provisioning:\s*#?\s*source:\s*".*"\s*#?\s*endpoint:\s*".*"\s*#?\s*dynamic_reprovisioning:\s*.*'
             $replacementContent = @(
                 'provisioning:',
                 '  source: ''external''',
-                "  endpoint: '$ExternalProvisioningEndpoint'")
+                "  endpoint: '$ExternalProvisioningEndpoint'",
+                "  dynamic_reprovisioning: $DynamicReprovisioning")
             $configurationYaml = ($configurationYaml -replace $selectionRegex, ($replacementContent -join "`n"))
```

Comment:
Do we need to remove manual provisioning section like what you did in line 1835 in this PR?

Reply:
Yes, you're right. Added that